cxotime/tests/test_cxotime_linspace.py

```diff
@@ -0,0 +1,173 @@
+import astropy.units as u
+import numpy as np
+import pytest
+
+from cxotime import CxoTime
+
+
+@pytest.mark.parametrize(
+    "start, stop, dt_max, expected_len, expected_values",
+    [
+        ("2000:001", "2000:001", 1 * u.day, 2, ["2000:001", "2000:001"]),
+        (
+            "2000:001",
+            "2000:005",
+            24 * u.hour,
+            5,
+            ["2000:001", "2000:002", "2000:003", "2000:004", "2000:005"],
+        ),
+        (
+            "2000:001",
+            "2000:005",
+            2880 * u.minute,
```

Comment:
The first two tests are exact integral divisors of 5 days, so they are basically the same test case. 

This test needs coverage of the case of a value that is not an integral divisor. There is no real need to change the units in these tests, you can trust that 1 * u.hour == 60 * u.minute == 3600 * u.s. (I had to open a terminal to see what `2880 / 60` was).